scipy/stats/_qmc.py

```diff
@@ -70,10 +70,11 @@ def scale(sample, l_bounds, u_bounds, reverse=False):
     sample : array_like (n, d)
         Sample to scale.
     l_bounds, u_bounds : array_like (d,)
-        Lower and upper bounds of transformed data. If `reverse` is True,
-        range of the original data to transform to the unit hypercube.
+        Lower and upper bounds (resp. ``a, b``) of transformed data. If
```

Comment:
Symbols ``a`` and ``b`` are not defined. Only :math:`a` and :math:`b` are defined. The comment was that you can either write 
```
resp. :math:`a`, :math:`b`
```
here, or you can rename `l_bounds`, `u_bounds` to `a` and `b` so that it is very obvious that they correspond with the math symbols above.

Reply:
Done